cmd/krew/cmd/index.go

```diff
@@ -47,6 +47,12 @@ each configured index in table format.`,
 		if err != nil {
 			return errors.Wrap(err, "failed to list indexes")
 		}
+
+		if len(indexes) == 0 {
+			klog.Info(indexoperations.NoIndexMessage)
+			return nil
+		}
```

Comment:
let's not do this here.

We can do that when user tries to query a plugin (`krew install`, `krew update`, `krew upgrade` etc) but there's no plugins installed.

Furthermore klog isn't good for this. we probably should use fmt.Fprintf or something like that with color pkg.

Reply:
Also I think this command should still print an "empty table".

We need to print the output properly to stderr so it doesn't get mixed up with other stuff.

Reply:
Do you mean it should still print:
```
INDEX  URL
```

Reply:
Yeah. That’s expected. Like krew list.